### ISSUE TITLE
paddleseg2.2可视化loss的问题 

### ISSUE THREAD

**Issue by Childishmi** [PaddlePaddle/PaddleSeg#1324]
您好，请问paddleseg2.2使用VisualDL，怎么可视化出test，val的loss曲线呢？用VisualDL我只能看到train/loss，evaluate/miou，evaluate/acc。

**Comment by michaelowenliu** [comment]
@Childishmi Hi, did you open --do_eval?

**Comment by Childishmi** [comment]
@michaelowenliu Hi, thanks for reply. I used.
我训练用的命令类似于“python PaddleSeg/train.py --config PaddleSeg/configs/unet/unet_cityscapes_1024x512_160k.yml --iters 15000 --learning_rate 0.1 --save_interval 200 --do_eval --use_vdl --save_dir output”；
然后使用“python PaddleSeg/predict.py --config PaddleSeg/configs/unet/unet_cityscapes_1024x512_160k.yml --model_path output/best_2200/model.pdparams --image_path destin_HA_clip/original_png/8.png”直接预测了；
（中间没有使用PaddleSeg/val.py）.

**Comment by Childishmi** [comment]
@michaelowenliu ,then after I open visualDL(http://localhost:8080/), it only appears **Train_loss** curve,
![image](https://user-images.githubusercontent.com/25951177/132457212-36cf9f4f-c548-464f-bd0d-71c403fa590a.png)
,I want to see **_test_loss_** or **_val_loss_** curves，I am newer，can you tell me how to do it please?
